lib/worker.js

```diff
@@ -0,0 +1,68 @@
+const {Task} = require('atom');
+const {default: PQueue} = require('p-queue');
+const uniqueString = require('unique-string');
+
+const queue = new PQueue({concurrency: 1});
```

Comment:
I'm unsure if this is the right approach.. I used `p-queue` to manage multiple lint jobs being triggered in quick succession. 

My only concern is fix-on-save jobs. If multiple are trigger quick succession, the end result might be broken? For that to happen, you'd need to be creating multiple lint errors, and pressing save, many times a second 😬 I wasn't able to do that..

Reply:
I don't think you need a queue for this. https://github.com/sindresorhus/p-limit is simpler.

---

> My only concern is fix-on-save jobs. If multiple are trigger quick succession, the end result might be broken? For that to happen, you'd need to be creating multiple lint errors, and pressing save, many times a second 😬 I wasn't able to do that..

Let's cross that bridge if it's actually an issue.